db/migrate/20210421143341_add_lead_school_to_schools.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLeadSchoolToSchools < ActiveRecord::Migration[6.1]
```

Comment:
@FriedSock has already created this file on his branch. Will use that once it's merged. This is temporary for testing purposes